integration/integration_test.go

```diff
@@ -117,7 +117,12 @@ func testMain(m *testing.M) int {
 	initCommands := []*command{}
 
 	if !*doNotDeploySPO {
-		initCommands = append(initCommands, deploySPO)
+		limit := false
+		if *k8sDistro == K8sDistroMinikubeGH {
+			limit = true
```

Comment:
Is it tested on Minikube outside of GitHub Actions? I wonder if Minikube on developers' computers might need the same limitation.

Reply:
However, I tried with Docker driver because it is the one used by the CI.

Reply:
Never mind. I had a problem during the tests. As we were expecting, the limitation is due to the limitation of the GH runners where Minikube is running and not on Minikube itself. Therefore, I will not make any changes to this part of the code.